current git master (b25deda) fails to build against latest granite master

I get the following error during compilation (it looks like maya still uses a widget that was removed from granite):

```
/builddir/build/BUILD/maya-calendar-0.4.0.2+rev1057/src/Widgets/ContractorButtonWithMenu.vala:47.45-47.78: error: The type name `Granite.Widgets.ToolButtonWithMenu' could not be found
    public class ContractorButtonWithMenu : Granite.Widgets.ToolButtonWithMenu {
                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
make[2]: *** [src/CMakeFiles/maya-calendar.dir/build.make:197: src/maya-calendar_valac.stamp] Error 1
make[1]: *** [CMakeFiles/Makefile2:345: src/CMakeFiles/maya-calendar.dir/all] Error 2
```

--- a/src/Widgets/ContractorButtonWithMenu.vala
+++ b/src/Widgets/ContractorButtonWithMenu.vala
@@ -44,13 +44,18 @@ namespace Maya.View.Widgets {
         }
     }
 
-    public class ContractorButtonWithMenu : Granite.Widgets.ToolButtonWithMenu {
+    public class ContractorButtonWithMenu : Gtk.MenuButton {
 
         private Gtk.FileChooserDialog filechooser;
 
         public ContractorButtonWithMenu (string tooltiptext) {
+            Object (
+                image: new Gtk.Image.from_icon_name ("document-export", Gtk.IconSize.LARGE_TOOLBAR),
+                tooltip_text: tooltiptext
+            );
 
-            base (new Gtk.Image.from_icon_name ("document-export", Gtk.IconSize.MENU), tooltiptext, new Gtk.Menu());
+            var menu = new Gtk.Menu ();
+
             try {
                 var contracts = Granite.Services.ContractorProxy.get_contracts_by_mime ("text/calender");
 
@@ -67,7 +72,8 @@ namespace Maya.View.Widgets {
             Gtk.MenuItem item = new Gtk.MenuItem.with_label(_("Export Calendar..."));
             item.activate.connect (savecal);
             menu.append (item);
-
+            menu.show_all ();
+            popup = menu;
         }
 
         private void savecal () {

